installer/vic-unified-installer.mk

```diff
@@ -31,13 +31,12 @@ VIC_ENGINE_BUNDLE := $(BIN)/vic-engine-bundle
 ovfenv := $(BIN)/ovfenv
 vic-ova-ui := $(BIN)/vic-ova-ui
 ova-webserver := $(BIN)/ova-webserver
-vic-machine-tarball := $(VIC_ENGINE_BUNDLE)/vic-engine-$(VER)-$(REV).tar.gz
+vic-tarball := $(VIC_ENGINE_BUNDLE)/vic-engine-$(VER)-$(REV).tar.gz
 ovfenv: $(ovfenv)
 vic-ova-ui: $(vic-ova-ui)
 ova-webserver: $(ova-webserver)
-vic-machine-tarball: $(vic-machine-tarball)
-# TODO(frapposelli): add vic-ui when ready
-vic-engine-bundle: $(appliance) $(bootstrap) $(vic-machine-linux) $(vic-machine-darwin) $(vic-machine-windows) $(vic-machine-tarball)
+vic-tarball: $(vic-tarball)
+vic-engine-bundle: $(appliance) $(bootstrap) $(vic-machine-linux) $(vic-machine-darwin) $(vic-machine-windows) $(vic-ui-linux) $(vic-ui-darwin) $(vic-ui-windows) $(vic-tarball)
```

Comment:
these `$(vic-ui-linux) $(vic-ui-darwin) $(vic-ui-windows)` targets don't exist, I guess they should be replaced by `$(vic-ui-plugins)`.